### ISSUE TITLE
Expose configuration options to the command line interface

### ISSUE THREAD

**Issue by Ian-Mint** [Ian-Mint/pong-underwater-rl#3]
Right now, we have a lot of hard-coded configuration options, from ball speed to learning rate. These should all be exposed to the command line so that we can run experiments without constantly changing the code.

**Comment by Yuanceli** [comment]
Done. May need revised for other hyperparameters later.
